### PR TITLE
(V2) networkd: bridge convert to jiffies

### PR DESCRIPTION
**Pull request by ssahani**

### CODE REVIEW

File: src/basic/time-util.c [hunk 1]
```diff
@@ -1122,3 +1122,17 @@ time_t mktime_or_timegm(struct tm *tm, bool utc) {
...
+        long r;
+
+        if (hz == 0) {
+                r = sysconf(_SC_CLK_TCK);
+
+                assert(r > 0);
+                hz = (unsigned long) r;
+        }
+
+        return (u + (USEC_PER_SEC / hz) - 1) / (USEC_PER_SEC / hz);
```
Comment (zonque) [inline src/basic/time-util.c]: Use `DIV_ROUND_UP`, please.

File: src/basic/time-util.c [hunk 2]
```diff
@@ -1122,3 +1122,17 @@ time_t mktime_or_timegm(struct tm *tm, bool utc) {
...
+        long r;
+
+        if (hz == 0) {
+                r = sysconf(_SC_CLK_TCK);
+
+                assert(r > 0);
+                hz = (unsigned long) r;
+        }
+
+        return DIV_ROUND_UP((u + (USEC_PER_SEC / hz) - 1), (USEC_PER_SEC / hz));
```
Comment (zonque) [inline src/basic/time-util.c]: No, `DIV_ROUND_UP` does that round up for you, hence the name. Just use `DIV_ROUND_UP(u, USEC_PER_SEC / hz)`.

Comment (ssahani) [inline src/basic/time-util.c]: thanks !

**Comment by zonque** [comment]
Ok, does it work, did you test the latest version? ;)

**Comment by ssahani** [comment]
yes ..

**Pull request merged**